src/en/myhentaigallery/src/eu/kanade/tachiyomi/extension/en/myhentaigallery/MyHentaiGallery.kt

```diff
@@ -107,17 +111,26 @@ class MyHentaiGallery : ParsedHttpSource() {
     // Details
 
     override fun mangaDetailsParse(document: Document): SManga {
-        return document.select("div.comic-header").let { info ->
+        return document.selectFirst("div.comic-header")!!.let { info ->
             SManga.create().apply {
-                title = info.select("h1").text()
+                title = info.selectFirst("h1")!!.text()
                 genre = info.select("div:containsOwn(categories) a").joinToString { it.text() }
-                artist = info.select("div:containsOwn(artists) a").text()
-                thumbnail_url = document.selectFirst(".comic-listing .comic-inner img")?.attr("src")
-                description = info.select("div:containsOwn(groups) a").let { groups ->
-                    if (groups.isNotEmpty()) "Groups: ${groups.joinToString { it.text() }}\n" else ""
-                }
-                description += info.select("div:containsOwn(parodies) a").let { groups ->
-                    if (groups.isNotEmpty()) "Parodies: ${groups.joinToString { it.text() }}" else ""
+                artist = info.select("div:containsOwn(artists) a").joinToString { it.text() }
+                thumbnail_url = document.selectFirst(".comic-listing .comic-inner img")?.absUrl("src")
+                description = buildString {
+                    info.select("div:containsOwn(groups) a")
+                        .takeIf { it.isNotEmpty() }
+                        ?.also { if (isNotEmpty()) append("\n\n") }
+                        ?.also { appendLine("Groups:") }
+                        ?.joinToString("\n") { "• ${it.text()}" }
```

Comment:
consider using markdown bullets (-) as main app now support markdown (at least in preview)